istio.json

```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://istio.io/",
+    "description": "Istio is an open platform-independent service mesh that provides traffic management, policy enforcement, and telemetry collection.",
+    "license": "Apache-2.0",
+    "version": "1.0.5",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/istio/istio/releases/download/1.0.5/istio-1.0.5-win.zip",
+            "hash": "ae50302ce5c45fdb06b3ffcdeb0d8f7e57ad74ff293f80740981a26d982b6982",
+            "extract_dir": "istio-1.0.5"
+        }
+    },
+    "bin": "bin\\istioctl.exe",
+    "checkver": {
```

Comment:
This should work

```json
"checkver": {
    "github": "https://github.com/istio/istio/" 
}
```

Reply:
Perfect! Switched to github for checkver